src/suites.rs

```diff
@@ -156,6 +156,8 @@ pub struct SupportedCipherSuite {
     /// in a deterministic and safe way.  GCM needs this,
     /// chacha20poly1305 works this way by design.
     pub explicit_nonce_len: usize,
+
+    pub(crate) hkdf_algorithm: ring::hkdf::Algorithm,
```

Comment:
Unless/until rustls returns a `Prk` directly for the QUIC secrets, can we get a getter for this?